plugins/default/lib/test_mongodb_health.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import unittest
+
+from mongodb_health import analize, parse_mongo_result, report
+
+
+class TestAnalize(unittest.TestCase):
+
+    def runTest(self):
+        self.assertEqual(analize(1), 0)
```

Comment:
We had discussed this before when @mikenairn brought this up and I thought the decision was to not take advantage of the constants in the tests, but I would agree they would improve readability greatly so happy to do it.